pkg/plugins/platform/local.go

```diff
@@ -186,6 +185,24 @@ func (l *LocalApp) ImportDB(imPath string) error {
 	return nil
 }
 
+// SiteStatus returns the current status of an application based on the web container.
+func (l *LocalApp) SiteStatus() string {
+	webContainer, err := l.FindContainerByType("web")
+	if err != nil {
+		return "not found"
```

Comment:
Only two outcomes (that I'm aware of) could cause `webContainer` to be empty.

1. Docker is not started, a log.fatal will be thrown by `FindContainerByType` with that as an error message.
2. The container wasn't found. In the context of list, this would be very curious indeed, since we build the actual list by looking at running/stopped docker containers. 

I think in this case we're probably OK as-is, since it really really should never happen anyway, unless a container is docker-rm'd between the `GetApps()` call and the `SiteStatus()` call in the listing. If we run into situations in the future where this isn't adequate, I'm all for changing, but it's hard to know what that looks like without context I think.